src/api/config/environments/development.rb

```diff
@@ -59,12 +59,3 @@
 
 CONFIG['frontend_host'] = "localhost"
 CONFIG['frontend_protocol'] = "http"
-
-require 'socket'
-fname = "#{Rails.root}/config/environments/development.#{Socket.gethostname}.rb"
-if File.exist? fname
-  STDERR.puts "Using local environment #{fname}"
-  eval File.read(fname)
-else
-  STDERR.puts "Custom development.#{Socket.gethostname}.rb not found - using defaults"
-end
```

Comment:
I often use it to set temporary config changes without. We have used it in the past for vagrant specific settings. I know @coolo uses it too somehow. Why do you want to remove it?